description: address format-security build error
author: Simon Ruderich <simon@ruderich.org>

--- a/src/jobs.c	2013-12-25 14:20:37.932958436 -0500
+++ b/src/jobs.c	2013-12-25 14:20:47.944958520 -0500
@@ -427,7 +427,7 @@
 				goto out;
 #endif
 		}
-		col = fmtstr(s, 32, strsignal(st));
+		col = fmtstr(s, 32, "%s", strsignal(st));
 #ifdef WCOREDUMP
 		if (WCOREDUMP(status)) {
 			col += fmtstr(s + col, 16, " (core dumped)");
